addons/l10n_mx/models/account_account.py

```diff
@@ -1,4 +1,11 @@
-from odoo import api, Command, models
+import re
+from odoo import _, api, Command, models
+from odoo.exceptions import RedirectWarning
+from odoo.tools import file_open
+
+PREFIX_COLUMN_VALUES = [line.split(',')[-1] for line in file_open('l10n_mx/data/account.group.template.csv').read().split('\n')]  # Getting last column value from each row
+SAT_CODES = [word[1:7] for word in PREFIX_COLUMN_VALUES if len(word) == 8]  # Filtering only "XXX.YY" values and removing quotes from inside the string
```

Comment:
```suggestion
SAT_CODES = {word.replace('"', '') for word in PREFIX_COLUMN_VALUES}  # Filtering only "XXX.YY" values and removing quotes from inside the string
```

Reply:
Does it impact performances to filter "really valid" values here?
I only need the values that follow the "XXX.YY" format, all others would be false, but I don't need to filter them out since the check of the accounts afterwards would report an issue if you're not using a "really valid" value.

Reply:
The replace is for lisibility, you can keep the condition if you want to filter out the length if you prefer. Switching to a set would gain more than the cost of generating a bigger one, but if you prefer to keep it smaller I have no objection. Maybe already matching the pattern here owuld make more sense?